packages/auth-passport/authService.js

```diff
@@ -121,12 +120,15 @@ PassportAuthService.prototype.logout = function() {
   });
 }
 
-angular.module('wfm.auth.passport').factory('authService', ['$http', '$window', '$mdDialog',
-  function($http, $window, $mdDialog) {
-    return new PassportAuthService($http, $window, $mdDialog);
+module.exports = function(isMobile) {
```

Comment:
We may move actual mouting this to demo mobile portal to get more control over differences.